src/constants.ts

```diff
@@ -15,9 +15,9 @@ export const DEFAULT_PIPELINE = "vib-pipeline.json"
 /**
  * Max waiting time for an execution graph to complete
  *
- * @default 90 minutes
+ * @default 6 hours
```

Comment:
6h is too much. I think we can have our default set to 2h but at the same time you need another global variable that sets the max time that GitHub will let your run an action on. That can be MAX_GITHUB_ACTION_RUN_TIME and should be set to 6 hours.

Reply:
done